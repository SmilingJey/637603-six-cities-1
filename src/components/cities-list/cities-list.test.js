import React from 'react';
import renderer from 'react-test-renderer';
import CitiesList from './cities-list.jsx';

const mockCities = [
  {
    name: `City1`
  },
  {
    name: `City2`
  },
  {
    name: `City3`
  },
  {
    name: `City4`
  },
  {
    name: `City5`
  },
  {
    name: `City6`
  },
];

it(`CitiesList snapshot`, () => {
  const tree = renderer.create(<CitiesList
    cities={mockCities}
    activeCity={mockCities[0]}
    onCityClick={jest.fn()}
  />).toJSON();
  expect(tree).toMatchSnapshot();
});
