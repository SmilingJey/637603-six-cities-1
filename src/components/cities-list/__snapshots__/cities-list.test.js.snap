// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CitiesList snapshot 1`] = `
<div
  className="cities tabs"
>
  <section
    className="locations container"
  >
    <ul
      className="locations__list tabs__list"
    >
      <li
        className="locations__item"
      >
        <a
          className="locations__item-link tabs__item
              tabs__item--active"
          href="#"
          onClick={[Function]}
        >
          <span>
            City1
          </span>
        </a>
      </li>
      <li
        className="locations__item"
      >
        <a
          className="locations__item-link tabs__item
              "
          href="#"
          onClick={[Function]}
        >
          <span>
            City2
          </span>
        </a>
      </li>
      <li
        className="locations__item"
      >
        <a
          className="locations__item-link tabs__item
              "
          href="#"
          onClick={[Function]}
        >
          <span>
            City3
          </span>
        </a>
      </li>
      <li
        className="locations__item"
      >
        <a
          className="locations__item-link tabs__item
              "
          href="#"
          onClick={[Function]}
        >
          <span>
            City4
          </span>
        </a>
      </li>
      <li
        className="locations__item"
      >
        <a
          className="locations__item-link tabs__item
              "
          href="#"
          onClick={[Function]}
        >
          <span>
            City5
          </span>
        </a>
      </li>
      <li
        className="locations__item"
      >
        <a
          className="locations__item-link tabs__item
              "
          href="#"
          onClick={[Function]}
        >
          <span>
            City6
          </span>
        </a>
      </li>
    </ul>
  </section>
</div>
`;
